package.json

```diff
@@ -1,6 +1,7 @@
 {
 	"dependencies": {
 		"@babel/parser": "^7.20.3",
-		"@babel/types": "^7.20.2"
+		"@babel/types": "^7.20.2",
+		"uuid": "^10.0.0"
```

Comment:
@priyanka-TL why do we have 2 different package.json?

Reply:
Removed